metaflow/plugins/kubernetes/kubernetes_client.py

```diff
@@ -1,8 +1,10 @@
+from concurrent.futures import ThreadPoolExecutor
```

Comment:
kubernetes_client is overloaded internally so we will have to ensure a symmetric change in the extensions